core/src/main/scala/kafka/admin/AdminUtils.scala

```diff
@@ -564,6 +564,11 @@ object AdminUtils extends Logging {
     topics.map(topic => fetchTopicMetadataFromZk(topic, zkUtils, cachedBrokerInfo))
   }
 
+  def fetchTopicMetadataFromZkWithProtocol(topics: Set[String], zkUtils: ZkUtils, protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT): Set[MetadataResponse.TopicMetadata] = {
```

Comment:
i had originally done
```
  def fetchTopicMetadataFromZk(topic: String, zkUtils: ZkUtils, protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT): MetadataResponse.TopicMetadata =
    fetchTopicMetadataFromZk(topic, zkUtils, new mutable.HashMap[Int, Broker], protocol)

  def fetchTopicMetadataFromZk(topics: Set[String], zkUtils: ZkUtils, protocol: SecurityProtocol = SecurityProtocol.PLAINTEXT): Set[MetadataResponse.TopicMetadata] = {
    val cachedBrokerInfo = new mutable.HashMap[Int, Broker]()
    topics.map(topic => fetchTopicMetadataFromZk(topic, zkUtils, cachedBrokerInfo, protocol))
  }

  private def fetchTopicMetadataFromZk(topic: String,
                                       zkUtils: ZkUtils,
                                       cachedBrokerInfo: mutable.HashMap[Int, Broker],
                                       protocol: SecurityProtocol): MetadataResponse.TopicMetadata = {

```

but i was getting `AdminUtils.scala:41: in object AdminUtils, multiple overloaded alternatives of method fetchTopicMetadataFromZk define default arguments.`